File: core/docmanager/audit_test.go
// core/docmanager/audit_test.go
// Tests unitaires pour AuditExistingScripts (Phase 2)

package docmanager

import "testing"

func TestAuditExistingScripts(t *testing.T) {
	result, err := AuditExistingScripts(".")
	if err != nil {
		t.Fatalf("Erreur AuditExistingScripts: %v", err)
	}
	if len(result.FilesFound) == 0 {
		t.Errorf("Aucun fichier trouvé lors de l'audit")
	}
}

File: scripts/audit.test.js
// scripts/audit.test.js
// Tests unitaires pour auditExistingScripts (Phase 2)

const { auditExistingScripts } = require('./audit');

test('auditExistingScripts finds files', () => {
  const result = auditExistingScripts('.');
  expect(result.filesFound.length).toBeGreaterThan(0);
});
